src/Sylius/Component/Core/Model/AdjustmentInterface.php

```diff
@@ -16,8 +16,10 @@
 interface AdjustmentInterface extends BaseAdjustmentInterface
 {
     // Labels for tax, shipping and promotion adjustments.
-    const TAX_ADJUSTMENT = 'tax';
-    const SHIPPING_ADJUSTMENT = 'shipping';
-    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
     const ORDER_ITEM_PROMOTION_ADJUSTMENT = 'order_item_promotion';
+    const ORDER_PROMOTION_ADJUSTMENT = 'order_promotion';
+    const ORDER_SHIPPING_PROMOTION_ADJUSTMENT = 'order_shipping_promotion';
```

Comment:
I only included the Sylius version of `Order::getShippingTotal` since it's now being called in the `OrderShipmentTaxesApplicator`. I thought it would be ok to bring it over as it is a dependency. I can drop it if you like and not worry about there being a call dependency there.